BREEZE.rst

```diff
@@ -696,85 +699,308 @@ This is the current syntax for  `./breeze <./breeze>`_:
         and setup. This file is automatically sourced when you enter the container. Database
         and webserver ports are forwarded to appropriate database/webserver so that you can
         connect to it from your host environment.
-  ****************************************************************************************************
+
+  Flags:
+
+  Run 'breeze flags' to see all  available applicable flags.
+
+
+  ####################################################################################################
+
+
+  Detailed usage for command: build-docs
+
   breeze [FLAGS] build-docs -- <EXTRA_ARGS>
 
         Builds airflow documentation. The documentation is build inside docker container - to
         maintain the same build environment for everyone. Appropriate sources are mapped from
         the host to the container so that latest sources are used. The folders where documentation
         is generated ('docs/build') are also mounted to the container - this way results of
         the documentation build is available in the host.
-  ****************************************************************************************************
+
+
+  ####################################################################################################
+
+
+  Detailed usage for command: build-image
+
   breeze [FLAGS] build-image -- <EXTRA_ARGS>
 
         Builds docker image (CI or production) without entering the container. You can pass
         additional options to this command, such as '--force-build-image',
         '--force-pull-image' '--python' '--use-local-cache'' in order to modify build behaviour.
         You can also pass '--production-image' flag to build production image rather than CI image.
-  ****************************************************************************************************
+
+  Flags:
+
+  -p, --python <PYTHON_MAJOR_MINOR_VERSION>
+          Python version used for the image. This is always major/minor version.
+          One of:
+
+                 3.6 3.7
+
+  -a, --install-airflow-version <INSTALL_AIRFLOW_VERSION>
+          If specified, installs airflow directly from PIP released version. This happens at
+          image building time in production image and at container entering time for CI image. One of:
+
+                 1.10.9 1.10.8 1.10.7 1.10.6 1.10.5 1.10.4 1.10.3 1.10.2 master v1-10-test
```

Comment:
```suggestion
                 1.10.10 1.10.9 1.10.8 1.10.7 1.10.6 1.10.5 1.10.4 1.10.3 1.10.2 master v1-10-test
```

Reply:
I think we should have a var of bash to get the all release support breeze here, we should not hard code it

Reply:
I got it, it just a doc, ignore it.

Reply:
But you are right I will add it to the list in breeze-complete.